xml/installation-installation-configure_lbaas.xml

```diff
@@ -724,338 +606,413 @@ rtt min/avg/max/mdev = 3.473/3.473/3.473/0.000 ms</screen>
+| project_id          | de095070f242416cb3dc4cd00e3c79f7     |
+| protocol            | HTTP                                 |
+| provisioning_status | PENDING_CREATE                       |
+| session_persistence | None                                 |
+| updated_at          | None                                 |
++---------------------+--------------------------------------+</screen>
+     <para>
+      Wait for it to become active
+     </para>
+<screen>&prompt.ardana;openstack loadbalancer pool show lb2_pool
++---------------------+--------------------------------------+
+| Field               | Value                                |
++---------------------+--------------------------------------+
+| admin_state_up      | True                                 |
+| created_at          | 2019-10-29T16:35:06                  |
+| description         |                                      |
+| healthmonitor_id    |                                      |
+| id                  | 75cd42fa-0525-421f-afaa-5de996267536 |
+| lb_algorithm        | ROUND_ROBIN                          |
+| listeners           | 65930b35-70bf-47d2-a135-aff49c219222 |
+| loadbalancers       | 2b0660e1-2901-41ea-93d8-d1fa590b9cfd |
+| members             |                                      |
+| name                | lb2_pool                             |
+| operating_status    | ONLINE                               |
+| project_id          | de095070f242416cb3dc4cd00e3c79f7     |
+| protocol            | HTTP                                 |
+| provisioning_status | ACTIVE                               |
+| session_persistence | None                                 |
+| updated_at          | 2019-10-29T16:35:10                  |
++---------------------+--------------------------------------+</screen>
    </step>
    <step>
     <para>
-     Create first member of the load balancing pool.
+     Put the private IPs of the tenant network for the first "webserver" in the pool
     </para>
-<screen>&prompt.ardana;neutron lbaas-member-create --subnet 6fc2572c-53b3-41d0-ab63-342d9515f514 \
-  --address 10.247.94.132 --protocol-port 80 lb1_pool
-+----------------+--------------------------------------+
-| Field          | Value                                |
-+----------------+--------------------------------------+
-| address        | 10.247.94.132                        |
-| admin_state_up | True                                 |
-| id             | 61da1e21-e0ae-4158-935a-c909a81470e1 |
-| protocol_port  | 80                                   |
-| subnet_id      | 6fc2572c-53b3-41d0-ab63-342d9515f514 |
-| tenant_id      | 4b31d0508f83437e83d8f4d520cda22f     |
-| weight         | 1                                    |
-+----------------+--------------------------------------+</screen>
-   </step>
-   <step>
     <para>
-     Create the second member.
+     Recall that we saved these when the VMs were first booted in the previous section:
```

Comment:
I ... don't know what this instruction is meant to indicate. Are you asking the using to recall the VMs using a certain command, or are you asking them to remember it?

If so, change the wording accordingly

Reply:
In the previous section the addresses for the VMs were saved in  the IP_LBVM1 and IP_LBVM2 bash variables. Use those variables or the or the literal addresses of the VMs

Reply:
the wording here has been changed around significantly